pkgs.nix

```diff
@@ -0,0 +1,15 @@
+let
```

Comment:
this one and `shell.nix` are top-level files, which ended up  by accident here?

Reply:
indeed :neutral_face: